cmds/rm/rm.go

```diff
@@ -69,10 +64,10 @@ func rm(files []string) error {
 			return err
```

Comment:
if err := f(file); err != nil {
  			return err
}
For this block of code, shouldn't it return the error if there was a problem with removing that specific file? the function f is assigned to os.Remove.